independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/InvocationBuilderImpl.java

```diff
@@ -197,8 +195,8 @@ public Response get() {
             throw new BlockingNotAllowedException();
         }
         try {
-            return c.get(readTimeoutMs, TimeUnit.MILLISECONDS);
```

Comment:
This is no longer needed (and is actually incorrect) as the timeout handling is performed by Vert.x